### DISCUSSION TITLE
Add '--platform=xxx' when build strimzi/base images for arm64 and ppc64le

### DISCUSSION THREAD

**Discussion by thuanghai** [strimzi/strimzi-kafka-operator#5511] [Ideas]
I have builded images for arm64 successfully. And I find a problem in the process of building 'strimzi/base'. The command 'make docker_build or make all' will build image for amd64 when using amd64 architecture host for building. So  I suggest add '--platform=' argument in its Dockerfile.
For example:
```
# Build for linux/arm64
FROM --platform=linux/arm64 registry.access.redhat.com/ubi8/ubi-minimal:latest
```
The 'linux/arm64' can be instead by a variable like TARGET_PLATFORM.
Also the environment variables below is needed. 
```
export DOCKER_BUILDX=buildx
export DOCKER_BUILD_ARGS="--platform linux/arm64 --load"
```

**Comment by scholzj** [comment]
The variables seem to be the same as what is described in the docs: https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/DEV_GUIDE.md#building-container-images-for-other-platforms-with-docker-buildx ... but I never needed to modify the FROM.

**Reply by thuanghai** [reply]
en. Yes, you are right! It is my misstake.
